modules/uploading-an-existing-notebook-file-from-a-git-repository-using-code-server.adoc

```diff
@@ -0,0 +1,30 @@
+:_module-type: PROCEDURE
+
+[id='uploading-an-existing-notebook-file-from-a-git-repository-using-code-server_{context}']
+= Uploading an existing notebook file from a Git repository by using code-server
+
+[role='_abstract']
+You can use the code-server user interface to clone a Git repository into your workspace to continue your work or integrate files from an external project.
```

Comment:
here you use "user interface" - do same in other places where you use "code-server workbench"

Reply:
that's my fault; previously when some of these sentences used to mention notebook-something, I was suggesting to just say "workbench" instead